DuckDuckGo/Tab/UserScripts/Subscription/SubscriptionErrorReporter.swift

```diff
@@ -36,6 +36,39 @@ enum SubscriptionError: Error {
          accountCreationFailed,
          activeSubscriptionAlreadyPresent,
          generalError
+
+    var localizedDescription: String {
+        switch self {
+        case .purchaseFailed:
+            return "Purchase process failed. Please try again."
```

Comment:
Just double checking... these aren't localised, but is that because none of the subscription strings are?

Reply:
1. We generally don't localise errors because they are used only in logs. The conformity to LocalisedError is just useful when in the logs we print any error received and the error's source can be Apple or us. Doing this allows us to just do `error.localisedDescription` on any kind of error and have a nice output
2. the localisation project is still a WIP